BETA_TESTING.md

```diff
@@ -4,13 +4,19 @@ This page describes the fastest way to test the latest version of DevTools with
 
 If you want to make code changes, follow [contributing guidance](https://github.com/flutter/devtools/blob/master/CONTRIBUTING.md).
 
-## Setup and start
+The steps were tested for Mac. They may require adjustments for other platforms.
+
+## Prerequisites
 
 1. [Configure](https://docs.flutter.dev/get-started/install) Dart or Flutter.
 
-2. In your terminal `cd` to a folder where you want to clone devtools, and that does not have subfolder `devtools` yet.
+2. Install `jq`: `brew install jq`
```

Comment:
Should this not rather be Homebrew and GitHub CLI as prerequisites assumed for the commands to work?

Reply:
why do we need jq?

Reply:
https://github.com/polina-c/devtools/blob/1902c331e5d2514937bdae0919941f5e357d795c/tool/latest_flutter_candidate.sh#L18